test/blackbox-tests/test-cases/melange/virtual-lib-transitive.t

```diff
@@ -0,0 +1,50 @@
+Test a case of virtual libraries where libraries depend on the virtual library
+transitively
+
+  $ mkdir -p foo vlib vlib_impl
+  $ cat > dune-project <<EOF
+  > (lang dune 3.13)
```

Comment:
Not sure if using the latest version of dune lang is desirable somehow.

Reply:
should be fine.